sycl/test-e2e/ESIMD/kmeans/kmeans.cpp

```diff
@@ -75,10 +75,10 @@ bool verify_result(Centroid4 *centroids4, // gpu centroids result
   int k = 0;
   int j = 0;
   for (auto i = 0; i < NUM_CENTROIDS_ACTUAL; i++) {
-    float errX = fabs(centroids4[j].x[k] - centroids[i].x) /
-                 max(fabs(centroids4[j].x[k]), fabs(centroids[i].x));
-    float errY = fabs(centroids4[j].y[k] - centroids[i].y) /
-                 max(fabs(centroids4[j].y[k]), fabs(centroids[i].y));
+    float errX = std::fabs(centroids4[j].x[k] - centroids[i].x) /
```

Comment:
Yes, this may affect users that have `using namespace sycl;` as the cmath builtins would have been taking precedence while the SYCL builtins weren't overloads. Arguably, this might also help users find bugs in their code as using cmath builtins in a SYCL kernel isn't guaranteed too work.

Reply:
ok thanks, seems legit to me and sounds like you guys thought about the impact